collector/Cargo.toml

```diff
@@ -6,7 +6,7 @@ edition = "2018"
 description = "Collects Rust performance data"
 
 [dependencies]
-clap = { version = "3.0.6", features = ["derive"] }
```

Comment:
Would it make sense for us to do `version = 3.2`?

Reply:
Yes, I changed it.